### ISSUE TITLE
Provide ability to filter the view on a cre page using the opencre URL

### ISSUE THREAD

**Issue by robvanderveer** [OWASP/common-requirement-enumeration#115]
Provide ability to filter the view on a cre page using the opencre URL.
At least by enumerating the standards that should be shown (eg. opencre.org/123-456&sources=NIST%20800-53%20v5,CWE
Plus: add the ability for the user to clear the filter, so that the full page is shown
Nice to have: ability to match only the beginning, so that version numbers can be left out: "sources=NIST*"

**Comment by northdpole** [comment]
due to the fact the field separators between standard sections and names are not always the same it's hard to translate "NIST 800-53 v5" means:
standard name: "NIST"
item: 800-53
version: v5

because then "Top10 2017 A3" becomes 
 standard name: "Top10"
item: 2017
version: A3

until we figure a way to have a consistent notation for separating fields in standards entries the simpler solution is to only provide the standard name and filter only by it.

**Comment by northdpole** [comment]
@robvanderveer  check this gif for a usage demo ![filters](https://user-images.githubusercontent.com/1583262/142472934-480cfc8c-56ae-40e6-ab74-072694993623.gif)

At the moment we can only filter on Standard names,
CREs are much harder to filter since we recursively fetch them, thus we need to wait until we have the full data to display before we can do so and with react this isn't trivial.
